File: Assignment22/C/Program1.c
// PS - Accept string from user and count number of capital letters in it

#include <stdio.h>

int CountCap(char *str)
{
    int iCount = 0;
    while (*str != '\0')
    {
        if ((*str >= 'A') && (*str <= 'Z'))
        {
            iCount++;
        }
        str++;
    }
    return iCount;
}

int main()
{
    char Arr[20];
    int iRet = 0;

    printf("Enter string \n");
    scanf("%[^'\n']s", Arr);

    iRet = CountCap(Arr);

    printf("Capital count of charqcters are %d \n", iRet);

    return 0;
}

File: Assignment33/Java/Program1.java
// PS - Write a java program which accept string ffrom user and return count of number of capital letters

import java.util.*;

class StringX
{
    public int CountCapital(String str)
    {
        char Arr[] = str.toCharArray();
        int iCount = 0;

        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if((Arr[iCnt] >= 'A') && (Arr[iCnt] <= 'Z'))
            {
                iCount++;
            }
        }
        return iCount;
    }
}

class Program1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter string");
        String content = sobj.nextLine();

        StringX obj = new StringX();
        int iRet = obj.CountCapital(content);

        System.out.println("Count of capital characters are : "+iRet);
    }
}